Add fade and height animations (#30)
infinum/ngx-nuts-and-bolts: libs/ngx-nuts-and-bolts/src/lib/animations/README.md

# Fade and Height animations

> This library includes two more commonly used animations (`fadeAnimation()` and `heightAnimation()`). For usage and general information about angular animations please consider visiting official [Angular animations docs](https://angular.io/guide/animations)

Reviewer: ```suggestion
This library includes two commonly used animations — (`fadeAnimation()` and `heightAnimation()`). For usage and general information about Angular animations, please consider visiting the official [Angular animations documentation](https://angular.io/guide/animations)
```
